### ISSUE TITLE
Expo crashes from Expo App (Published) Or Hard Reload Android

### ISSUE THREAD

**Issue by zocheyado** [expo/expo#3148]
Environment:
  OS: macOS 10.14
  Node: 10.4.1
  Yarn: 1.6.0
  npm: 6.1.0
  Watchman: Not Found
  Xcode: Xcode 10.1 Build version 10B61
  Android Studio: Not Found

Packages: (wanted => installed)
  expo: ^27.0.1 => 27.1.1
  react: 16.3.1 => 16.3.1
  react-native: ~0.55.2 => 0.55.4

Working on iOS but not Android

Trying to track down a bug and having some issues. If I run my app via Expo (dev mode) it runs fine no issues, if I run app in production mode in expo, works fine. However if I publish the app and open from link or qr code (after publish) it crashes on start. In addition if I edit something in my code the hot reload works fine, but if I drag down from the top and press the expo refresh button it crashes (no errors).

Basically everything works fine on android until I publish ( the publishing has no errors ). Or if I refresh through the expo app. Shake and refresh on iOS works fine, opening expo published app on iOS works fine.

Any idea how I can get some errors to get a direction on why this is happening ?

**Comment by brentvatne** [comment]
hi! please update the expo client on android, it should be resolved now

**Comment by zocheyado** [comment]
Oh that worked! Perfect. Was there a PR that explained what was wrong? Just so I know.

**Comment by brentvatne** [comment]
https://github.com/expo/expo/commit/b4968c50a2eb575d2415ea3add48d3dfacad2800
